### PR TITLE
Feature/file sharing

### PR DESCRIPTION
**Pull request by athar-shayan**
Update backend with accepting other than text message.

### CODE REVIEW

File: socket.js [hunk 1]
```diff
@@ -1,6 +1,16 @@
 import { Server } from "socket.io";
 import { updateSocketId } from "./Controller/user.js";
+import { newMessage } from "./Controller/message.js";
 
+import { v2 as cloudinary } from "cloudinary";
+
+cloudinary.config({
+  cloud_name: "dubwggg4a",
+  api_key: "431517811498724",
+  api_secret: "eoejYyKRwg1jg6EBHzh2evyjkEE",
```
Comment (imran-munawar) [inline socket.js]: @athar-shayan I've previously mentioned in your previous pull requests to store sensitive information such as `cloud_name`, `api_key`, and `api_secret` in your .env files. Please make sure to move these credentials to the .env files.

**Review by imran-munawar — CHANGES_REQUESTED** [review]

File: socket.js [hunk 2]
```diff
@@ -1,6 +1,16 @@
 import { Server } from "socket.io";
 import { updateSocketId } from "./Controller/user.js";
+import { newMessage } from "./Controller/message.js";
 
+import { v2 as cloudinary } from "cloudinary";
+
+cloudinary.config({
+  cloud_name: "dubwggg4a",
```
Comment (asad-shayansolution) [inline socket.js]: Put these credentials into environment file

File: socket.js (hunk 1 above)
Comment (athar-shayan) [reply socket.js]: Done.

Comment (imran-munawar) [reply socket.js]: @athar-shayan Currently, you are accessing the values directly from the .env file. Sir Arslan suggested creating a constant.js file and importing these constants into that file abd then use it.

**Review by imran-munawar — APPROVED** [review]

File: constant/constant.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import dotenv from "dotenv";
+
+// dot evn package
+dotenv.config();
```
Comment (asad-shayansolution) [inline constant/constant.js:4]: move this config function to server file

File: package.json [hunk 4]
```diff
@@ -11,12 +11,14 @@
...
+    "cloudinary": "^1.40.0",
     "cors": "^2.8.5",
     "dotenv": "^16.3.1",
     "express": "^4.18.2",
     "mongodb": "^5.8.1",
     "mongoose": "^7.4.4",
     "nodemon": "^3.0.1",
-    "socket.io": "^4.7.2"
+    "socket.io": "^4.7.2",
+    "start": "^5.1.0"
```
Comment (asad-shayansolution) [inline package.json]: remove start library

File: server.js [hunk 5]
```diff
@@ -1,15 +1,13 @@
 import express from "express";
 import cors from "cors";
 import mongoose from "mongoose";
-import dotenv from "dotenv";
 import userRouter from "./Routes/user.js";
 import conversationRouter from "./Routes/conversation.js";
 import messageRouter from "./Routes/message.js";
 import { createServer } from "http";
 import socketio from "./socket.js";
+import { MONGO_DB, PORT } from "./constant/constant.js";
```
Comment (asad-shayansolution) [inline server.js:10]: enable this line of code for env data

**Review by asad-shayansolution — CHANGES_REQUESTED** [review]
make suggested changes

**Review by asad-shayansolution — APPROVED** [review]

File: constant/constant.js (hunk 3 above)
Comment (athar-shayan) [reply constant/constant.js:4]: config function is necessity to get data from .env file.

**Pull request merged**